lib/nexmo_markdown_renderer/filters/i18n/smartling/frontmatter_filter.rb

```diff
@@ -4,14 +4,30 @@ module I18n
       module Smartling
         class FrontmatterFilter < Banzai::Filter
           def call(input)
-            input.gsub(/\A\*\*\* \*\* \* \*\* \*\*\*\n*([^-]*)\n*-+/m) do |_frontmatter|
-              front = $1.gsub(/`(.*):`(.*)/) do |_config|
+            input.gsub(/\A\*\*\* \*\* \* \*\* \*\*\*\n*(.*\n)!?(----+|\*\*\* \*\* \* \*\* \*\*\*)\n*/m) do |_frontmatter|
+              front = $1.gsub(/`products: (.*)`\n\n/) do |products|
+                "products: #{$1}\n\n"
+              end
+
+              front = front.gsub(/`(.*):`(.*)/) do |_config|
                 "#{$1}: #{$2}"
               end
+
+              front = front.gsub(/languages: \n\n(.*)\n/m) do |_languages|
+                languages = $1.split("\n\n").map do |lang|
```

Comment:
So we need to make sure there is two line breaks in the metadata? Is that standard now?

Reply:
that's how it is comming from smartling :/

Reply:
That's what I was wondering, now it makes perfect sense